package-lock.json

```diff
@@ -15975,7 +15975,7 @@
     },
     "toml": {
       "version": "3.0.0",
-      "resolved": "https://registry.npmjs.org/toml/-/toml-3.0.0.tgz",
+      "resolved": "https://repo.sonatype.com/repository/npm-all/toml/-/toml-3.0.0.tgz",
```

Comment:
@DarthHater  minor: Still catching up on the whirlwind of activity, but I noticed a similar change to the URL in package-lock.json, I'm pretty sure due to npm changes required to hit our `npm-all` repo. Do we care if these are go into the package-lock.json files?

Reply:
No we absolutely need to change these back to registry.npmjs.org, the builds are passing on CircleCI because it's got cached deps (which is great).